desktop/fileManager.js

```diff
@@ -58,7 +74,7 @@ class FileManager {
     }
   };
 
-  static writeFile = async ({ browserWindow, event, fileName, fileContent, showDialog }) => {
+  static writeFile = async ({ browserWindow, event, fileName, fileContent, showDialog, saveToDocumentsFolder }) => {
```

Comment:
Is there a function that calls this function with param showDialog set to true?

Reply:
Not that I'm aware of. Would you like me to change anything here?

Reply:
The right way to make changes to unknown code is to check first all possible use cases of this code. In this case you changed single function call with "showDialog: true" to "showDialog: false" and didn't make any effort to understand what is this param and what is it's usage. So hopefully now you will study the function's logic and make proper changes to it to fit new requirements